### DISCUSSION TITLE
ESP32 Ethernet RMII clock configuration

### DISCUSSION THREAD

**Discussion by andriyhrabchak** [zephyrproject-rtos/zephyr#67265] [Q&A]
I am working with integration of custom ESP32 based hardware. Namely, I am trying to run Ethernet and I need to change default ESP-IDF settings (namely CONFIG_ETH_RMII_CLK_OUTPUT and CONFIG_ETH_RMII_CLK_OUT_GPIO), but I do not know how to propagate them correctly with existing Zephyr setup. I am using Windows and had set Zephyr development environment as stated here https://docs.zephyrproject.org/3.4.0/develop/getting_started/index.html

**Comment by bbilas** [comment]
Some time ago I was working on changes related to external clk output for ETH phy, check https://github.com/zephyrproject-rtos/zephyr/pull/65759.

**Comment by andriyhrabchak** [comment]
Great! After updating to new zephyr I am able to setup desired clock option, but suddenly my phy started to throw the error like "[00:00:02.107,000] <err> phy_mii: No PHY found at address 1". I am shure that in previous version phy was not complain ...
